src/database/seeds/Users.ts

```diff
@@ -13,6 +13,7 @@ const sampleParticipant = {
   status: ParticipantStatus.AwaitingSigning,
   allowSharing: true,
   location: 'Sydney',
+  types: [],
```

Comment:
Why this? It's optional and the thing it's used for uses a separate parameter for type.

Reply:
removed